frontend/package.json

```diff
@@ -27,26 +27,27 @@
     "format": "prettier --write \"./**/*.ts\" \"./**/*.tsx\" \"./**/*.scss\"",
     "lint:goal": "eslint --config ./.eslintrc.goal.js --ext .json,.js,.ts,.jsx,.tsx ./src ",
     "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
-    "test": "run-s test:lint test:unit test:type-check test:integration-ci",
+    "test": "run-s test:lint test:type-check test:unit test:integration-ci test:cypress-ci",
     "test:dev": "run-s test:lint test:unit test:accessibility test:integration",
     "test:fix": "eslint --ext .js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .js,.ts,.jsx,.tsx ./src",
     "test:type-check": "tsc --noEmit",
     "test:jest": "jest",
     "test:unit": "npm run test:jest",
     "test:accessibility": "test-storybook --verbose",
+    "test:cypress-ci": "npx concurrently -k -s first \"npm run cypress:server:build && npm run cypress:server\" \"npx wait-on tcp:127.0.0.1:9001 && npm run cypress:run:mock\"",
```

Comment:
Similar to `test:integration-ci`. We run the cypress build followed by the cypress server and wait for the server to be up before running cypress. The whole command ends when cypress finishes.